src/com/company/repin/MyArrayList.java

```diff
@@ -0,0 +1,102 @@
+package com.company.repin;
+
+
+public class MyArrayList<T> implements MyList<T> {
+    private Object[] array;
+    private int counter = 0;
+
+    public MyArrayList() {
+        array = new Object[0];
+    }
+
+
+    @Override
+    public T get(int i) {
+        for (int j = 0; j < array.length; j++) {
+            if (j == i) {
+                return (T) array[j];
+            }
+        }
+        return null;
+    }
+
+    @Override
+    public void add(T t) {
```

Comment:
and in add method just check if your counter == size then increase your array size